docker-app/qfieldcloud/core/tests/test_project.py

```diff
@@ -804,3 +804,91 @@ def test_seamless_projects_fetching_with_public_ones(self):
         self.assertEqual(data[3]["owner"], "user1")
         self.assertEqual(data[3]["name"], "public_project_of_user1")
         self.assertTrue(data[3]["is_public"])
+
+    def test_restricted_data_last_updated_at_on_file_upload(self):
+        """Test that restricted_data_last_updated_at is updated when restricted files are uploaded."""
+        from io import StringIO
+
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token1.key)
+
+        # Create a project
+        project = Project.objects.create(
+            name="test_restricted_data", is_public=False, owner=self.user1
```

Comment:
```suggestion
            name="test_restricted_data", owner=self.user1
```

No need to pass this explicitly, unless you want to change the default.